### PR TITLE
Fixed displaying RC scope.

### PR DESCRIPTION
**Pull request by Daemach**
The RC scope wasn't displaying properly.

It is now.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 /tests/results
 /tests/resources/app/coldbox
 /node_modules
-/modules
\ No newline at end of file
+/modules
+.vscode/settings.json
```
Comment (elpete) [inline .gitignore:6]: I'd suggest adding this to your own global gitignore file so you don't have to add it to every repo you contribute to. https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

**Comment by Daemach** [comment]
You had 2 blocks named frameworksnapshot_scope.  The RC scope was not displaying when clicking on the filter button before making this change.

**Comment by Daemach** [comment]
More info here:  https://www.screencast.com/t/FNVrQwxYB

File: .gitignore (hunk 1 above)
Comment (Daemach) [reply .gitignore]: Good suggestion.  I'm pretty new to all of this and I appreciate your patience with me.

**Comment by elpete** [comment]
Thanks for this!

**Comment by elpete** [comment]
Published as v2.2.1

**Pull request merged**